taxcalc/tests/test_calculate.py

```diff
@@ -89,133 +69,130 @@ def test_sequence():
-    user_mods = """{"1991": { "_STD_Aged": [[1500, 1250, 1200, 1500, 1500, 1200 ]],
-                     "_STD_Aged_cpi": false}}"""
-
-    calc2 = calculator(params, puf, mods=user_mods, _II_em_cpi=False,
-                       _II_em=np.array([4000]))
+def test_make_Calculator_with_policy_reform2():
+    # create a Parameters object and apply a policy reform
+    params2 = Parameters()
+    reform2 = {2013: {'_II_em': np.array([4000]), '_II_em_cpi': False,
+                      '_STD_Aged': [[1600, 1300, 1300, 1600, 1600, 1300]],
+                      "_STD_Aged_cpi": False}}
+    params2.implement_reform(reform2)
+    # create a Calculator object using this policy-reform
+    puf = Records(data=TAX_DTA, weights=WEIGHTS, start_year=2009)
+    calc2 = Calculator(params=params2, records=puf)
+    # check that Parameters object embedded in Calculator object is correct
+    assert calc2.current_year == 2013
     assert calc2.params.II_em == 4000
     assert_array_equal(calc2.params._II_em, np.array([4000] * 12))
-    exp_STD_Aged = [[1500, 1250, 1200, 1500, 1500, 1200]] * 12
+    exp_STD_Aged = [[1600, 1300, 1300, 1600, 1600, 1300]] * 12
     assert_array_equal(calc2.params._STD_Aged, np.array(exp_STD_Aged))
-    assert_array_equal(calc2.params.STD_Aged, np.array([1500, 1250, 1200, 1500,
-                                                        1500, 1200]))
-
-
-def test_make_Calculator_user_mods_as_dict():
-
-    # Create a Params object
-    params = Parameters(start_year=1991, inflation_rates=irates)
-
-    # Create a Public Use File object
-    puf = Records(tax_dta)
-
-    user_mods = {1991: {"_STD_Aged": [[1400, 1200, 1000, 1000, 1000, 1000]]}}
-    user_mods[1991]['_II_em'] = [3925, 4000, 4100]
-    user_mods[1991]['_II_em_cpi'] = False
-    calc2 = calculator(params, puf, mods=user_mods)
-    assert calc2.params.II_em == 3925
-    exp_II_em = [3925, 4000] + [4100] * 10
-    assert_array_equal(calc2.params._II_em, np.array(exp_II_em))
     assert_array_equal(calc2.params.STD_Aged,
-                       np.array([1400, 1200, 1000, 1000, 1000, 1000]))
-
-
-def test_make_Calculator_with_reform_after_first_year():
-    fyr = 2008
-
-    # specify reform in in 2010
-    user_mods = {2010: {"_STD_Aged": [[1501, 1202, 1502, 1203, 1504, 1204]]}}
-    user_mods[2010]['_II_em'] = [5000, 6000]
-    user_mods[2010]['_II_em_cpi'] = False
-
-    # create a Parameters object
-    irates = {2008: 0.021, 2009: 0.022, 2010: 0.021, 2011: 0.022}
-    params = Parameters(start_year=fyr, inflation_rates=irates,
-                        num_years=len(irates))
-    params.implement_reform(user_mods)
-
-    # create a Records object
-    tax_dta2 = pd.read_csv(tax_dta_path, compression='gzip')
-    rec = Records(tax_dta2, start_year=fyr)
-
-    # create Calculator object with params and rec objects
-    calc = Calculator(params, rec)
-
+                       np.array([1600, 1300, 1300, 1600, 1600, 1300]))
+
+
+def test_make_Calculator_with_policy_reform3():
```

Comment:
this test exercises a multi-year reform, so I suggest renaming to `test_make_Calculator_with_mulityear_reform`. This will make explicit how it differs from `test_make_Calculator_with_policy_reform`, so as a developer works on the repo, he/she will immediately see what is broken if one or both of the tests fail.